kubernetes-AKS/ingress.yaml

```
@@ -0,0 +1,36 @@
+  annotations:
+    # Rate limiting: 3 requests per minute (1 request every 20 seconds)
+    nginx.ingress.kubernetes.io/rate-limit: "3"
+    nginx.ingress.kubernetes.io/rate-limit-window: "1m"
+    # Rate limit based on client IP
+    nginx.ingress.kubernetes.io/rate-limit-key: "$remote_addr"
+    # Custom error page for rate limit exceeded
+    nginx.ingress.kubernetes.io/custom-http-errors: "429"
+spec:
+  ingressClassName: nginx
+  rules:
+  - host: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)